chainguard-dev/edu
# Initial rumble support for images listed in data/rumble.json

## Type of change
platform

### What should this PR do?
This PR adds support for embeding rumble bar charts on pages. It only embeds on pages that are listed in the `data/rumble.json` file, since many Chainguard images don't have a corresponding external image to compare with.

### Why are we making this change?
Showing CVE counts with commonly pulled images is a win for everyone!

### What are the acceptance criteria? 
Data should show up in pages that are listed in `data/rumble.json`. For example rabbitmq:

![Screenshot 2023-05-25 at 11 41 53 AM](https://github.com/chainguard-dev/edu/assets/328553/8f56c33d-d7f7-4b15-ba2a-2a43c86b6125)

### How should this PR be tested?
Run hugo locally and visit any image overview page that has an entry in `data/rumble.json` should show the bar chart.
